.buildkite/pipeline.yml

```diff
@@ -431,9 +429,42 @@ steps:
           slurm_mem: 20GB
           slurm_gpus: 1
 
-
-
-      - wait
+  # - group: "Bash scripts"
+
+  #   steps:
+
+  #     - label: "Submit and Monitor sbatch Job on Caltech HPC"
```

Comment:
Maybe let's also add the comment on why we are doing this. (We want to check that (1) the script can be succesfully submitted, (2) it runs successfully